### PR TITLE
Set `trailingComma` default value to `es5`

### PR DESCRIPTION
**Pull request by fisker**
- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: tests/angular_component_examples/jsfmt.spec.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
+run_spec(__dirname, ["typescript"], { trailingComma: "none" });
 run_spec(__dirname, ["typescript"]);
```
Comment (fisker) [inline tests/angular_component_examples/jsfmt.spec.js]: I change the test orders to help review, snap chang should only be `options` part

File: tests/css_scss/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -1987,7 +1987,7 @@ $my-map: (
   "foo": 1,
   // Comment
     "bar": 2,
-  // Comment
+  // Comment,
```
Comment (fisker) [inline tests/css_scss/__snapshots__/jsfmt.spec.js.snap]: this should be a scss bug, need fix with another PR

Comment (fisker) [reply tests/css_scss/__snapshots__/jsfmt.spec.js.snap]: issue: https://github.com/prettier/prettier/issues/6911

File: tests/angular_interpolation/__snapshots__/jsfmt.spec.js.snap [hunk 3]
```diff
@@ -15,7 +15,7 @@ printWidth: 80
   advancedSearchService.patientInformationFieldsRow2 &&
     advancedSearchService.patientInformationFieldsRow2.indexOf(
       advancedSearchService.formElementData.customFieldList[i].customFieldType
-    ) !== -1
+    ) !== -1,
```
Comment (fisker) [inline tests/angular_interpolation/__snapshots__/jsfmt.spec.js.snap]: issue #6912

File: src/language-js/options.js [hunk 4]
```diff
@@ -76,16 +76,17 @@ module.exports = {
...
     ],
     description: "Print trailing commas wherever possible when multi-line.",
     choices: [
-      { value: "none", description: "No trailing commas." },
       {
         value: "es5",
         description:
           "Trailing commas where valid in ES5 (objects, arrays, etc.)"
       },
+      { value: "none", description: "No trailing commas." },
```
Comment (lydell) [inline src/language-js/options.js]: Why is this moved?

Comment (fisker) [reply src/language-js/options.js]: playground shows none above es5

**Comment by fisker** [comment]
waiting for two blockers #6918 and #6912

**Comment by thorn0** [comment]
https://github.com/prettier/prettier/issues/6888#issue-520308285

File: src/language-js/options.js (hunk 4 above)
Comment (ExE-Boss) [reply src/language-js/options.js]: I’d&nbsp;prefer if&nbsp;the&nbsp;order wasn’t changed, as&nbsp;that&nbsp;way, `es5`&nbsp;is&nbsp;still implied to&nbsp;be&nbsp;between `none`&nbsp;and&nbsp;`all`.

**Comment by lydell** [comment]
Changing the default of `trailingComma` was proposed in the original 2.0 issue #3503 (after having being requested in other issues) and I’ve seen 0 mentions against it.

File: src/language-js/options.js (hunk 4 above)
Comment (fisker) [reply src/language-js/options.js]: @ExE-Boss why? after changing default to `es5`, shouldn't we put it on the top?

**Comment by fisker** [comment]
accidentlly force-pushed a wrong commit, auto closed, please reopen

**Comment by lydell** [comment]
The "Reopen" button is disabled for me. Its tooltip says: "There are no new commits on the fisker:trailing-comma-es5 branch."

**Comment by fisker** [comment]
new PR #6963
